src/modules/partners/partner_test.ts

```diff
@@ -0,0 +1,19 @@
+import request from 'supertest'
+import app from '../../server'
+
+describe('tests partners', () => {
+  it('test success findAll', async () => {
+    const response = await request(app)
+      .get('/v1/partners')
+
+    expect(response.statusCode).toEqual(200)
+  })
+
+  it('test success findAll with requestQuery', async () => {
+    const response = await request(app)
+      .get('/v1/partners')
+      .query({ name: 'test' })
+
+    expect(response.statusCode).toEqual(200)
+  })
```

Comment:
Please add tests with asserting the response, e.g. if we have 1 partner in database, then the json response should contain that 1 particular partner. If we have 2 partner in database, one named "dummy" and one named "test", then the json response should only contain the partner named "test".

Reply:
means adding seeders ?

Reply:
hmm not a seeder I think. Seeding database means it will remain there, right? I think it'd be better if we just create it on the test and test it right away.

Reply:
Oh, are we using the same database when running the test?

Reply:
yes, we use API test for testing, database is running every run test

Reply:
okay, I think I need further information about this, maybe in the sync meeting later

Reply:
yes TBD :smile: